src/packages/excalidraw/CHANGELOG.MD

```diff
@@ -7,11 +7,14 @@ The change should be grouped under one of the below section and must contain PR
 - Chore: Changes for non src files example package.json.
 - Improvements: For any improvements.
 - Refactor: For any refactoring.
+
+Please add the latest features to the top!
```

Comment:
```suggestion
The latest change should always be on top under any of the sections!
```

Reply:
mm, idk about this one, it sounds nicer when you add please

Reply:
@ad1992 lemme know if you prefer the second one for sure

Reply:
If anything we should just remove the `!`, otherwise the original is OK.

Reply:
@mwufi sure let's make it `Please add the latest change on the top under the correct section!` so it's valid for all types of changes and not just features.

Reply:
Props on the last version, but let's not yell at people as @lipis suggested! :)

Reply:
yep :) I missed out the suggestion while reviewing.